src/blog/2024-09-29-transformer-alpha.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Oxc Transformer Alpha
+outline: deep
+authors:
+  - boshen
+  - Dunqing
+  - overlookmotel
+---
+
+<AppBlogPostHeader />
+
+We are excited to announce an alpha release for Oxc transformer (a.k.a transpiler).
+
+This release contains two major features:
+
+1. Transforming TypeScript and React JSX files to ESNext.
+2. [TypeScript Isolated Declarations DTS Emit](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-5-5.html#isolated-declarations) (`tsc --isolatedDeclarations`).
+
+In this alpha stage, we recommend to experiment these features to significantly speed up continuous integration time.
+
+[Our Benchmark](https://github.com/oxc-project/bench-transformer) shows that:
+
+- For transform, oxc is 4x faster than swc, uses 20% less memory, and is 35 MB smaller in package size (from swc's 37MB).
+- For transform, oxc is 40x faster than babel, uses 70% less memory, and is 19 MB smaller with 168 npm packages less to install.
+- For react development + refresh, oxc is 6x faster than swc and 70x faster than Babel.
+- For tsc's isolated declarations dts emit, oxc is 45x faster on ordinary files, and 20x faster on larger files.
```

Comment:
Make capitalization consistent and clarify comparisons (well clearer in my opinion anyway):

```suggestion
- Transform: Oxc is 4x faster than SWC, uses 20% less memory, and has far smaller package size (2 MB vs SWC's 37 MB).
- Transform: Oxc is 40x faster than Babel, uses 70% less memory, and is 19 MB smaller, with only 2 npm packages to install vs Babel's 170.
- React development + React Refresh: Oxc is 6x faster than SWC, and 70x faster than Babel.
- TS isolated declarations `.d.ts` emit: Oxc is 45x faster than TSC on typical files, 20x faster on larger files.
```